crates/ruff_linter/src/rules/pydoclint/rules/check_docstring.rs

```diff
@@ -645,15 +646,14 @@ pub(crate) fn check_docstring(
     convention: Option<Convention>,
 ) {
     let mut diagnostics = Vec::new();
-    let Definition::Member(member) = definition else {
+
+    // Only check function docstrings.
+    let Some(function_def) = definition.as_function_def() else {
         return;
     };
 
-    // Only check function docstrings.
-    if matches!(
```

Comment:
Is this check now covered by is_stub?

Reply:
It's covered by the 

```rs
let Some(function_def) = definition.as_function_def() else {
    return;
};
```

immediately above